src/Plugin/EntityBrowser/Display/IFrame.php

```diff
@@ -165,6 +165,9 @@ public function selectionCompleted(array $entities) {
    *   Response event.
    */
   public function propagateSelection(FilterResponseEvent $event) {
+    if (empty($this->entities)) {
```

Comment:
This is snuck in from #72, as I still get it even with newer versions of entity browser. As I'm not using AJAX anymore with filters I can remove this code, if necessary.